modules/gdscript/gdscript_editor.cpp

```diff
@@ -499,8 +499,14 @@ struct GDScriptCompletionIdentifier {
 
 static void _get_directory_contents(EditorFileSystemDirectory *p_dir, Set<String> &r_list) {
 
+#ifdef TOOLS_ENABLED
```

Comment:
This code is already enclosed in a `#if defined(TOOLS_ENABLED)` (line 472), so it's not needed here.